### DISCUSSION TITLE
[Question]: Free space

### DISCUSSION THREAD

**Discussion by nulkathos** [blakeblackshear/frigate#10156] [Ask A Question] [answered]
### What is your question:

I think that soon I will run out of space .
I've ordered an hd but I think will not be delivered before 3-4 days
there is a way to clean/remove the oldest records?

**Comment by NickM-27** [comment, marked as answer]
by default frigate will clear up space if there is not enough room to continue recording

**Reply by nulkathos** [reply]
thank you Nick
